docker-compose.yml

```diff
@@ -104,9 +104,14 @@ services:
       DOCKER: 'true'
 
       #
-      # Enables logging to stdout instead of file.
+      # Enables logging to stdout instead of file. Override this in .env
       #
-      RAILS_STDOUT_LOGGING: "true"
+      RAILS_STDOUT_LOGGING: "${RAILS_STDOUT_LOGGING:-true}"
```

Comment:
This line change shouldn't be necessary. Any ENV defined in docker-compose can be overridden by the .env.
These kinds of defaults inside of `${...}` are only necessary when the variable is used in other parts of the file (not the environment section.

Reply:
Never mind I was wrong. This is needed.